pkg/service/schema.go

```diff
@@ -0,0 +1,83 @@
+package service
+
+// ParameterSchemas is a set of optional JSONSchemas that describe
+// the expected parameters for creation and update of instances and
+// creation of bindings.
+type ParameterSchemas struct {
```

Comment:
I'm worried about how close the name of this is to that of another type-- `ParametersSchema`.

Any thoughts about renaming this as `PlanSchema` (singular, too)? I feel that accurately reflect what this is

Reply:
Sure! That works for me.

Reply:
Actually, I take that back.

Reply:
The name was tricky to wrap my head around, but I get it now.

Reply:
Reconsidering the naming-- I think `planSchema` is good after all. Let's not export this if we don't need to.